src/sentry/static/sentry/app/views/alerts/rules/details/metricChart.tsx

```diff
@@ -1,145 +1,250 @@
 import React from 'react';
+import color from 'color';
 import moment from 'moment';
 
+import Graphic from 'app/components/charts/components/graphic';
 import MarkLine from 'app/components/charts/components/markLine';
 import LineChart, {LineChartSeries} from 'app/components/charts/lineChart';
 import space from 'app/styles/space';
-import {Series} from 'app/types/echarts';
+import {ReactEchartsRef, Series} from 'app/types/echarts';
 import theme from 'app/utils/theme';
-import {Trigger} from 'app/views/settings/incidentRules/types';
+import {IncidentRule, Trigger} from 'app/views/settings/incidentRules/types';
 
 import {Incident} from '../../types';
 
+const X_AXIS_BOUNDARY_GAP = 15;
+
 type Props = {
   data: Series[];
+  rule?: IncidentRule;
```

Comment:
do we need to pass in the whole rule if we only use it for `rule.dateModified`?

Reply:
Yeah, that makes sense